README.md

```diff
@@ -146,9 +146,12 @@ This document discusses how to build mlpack from source. These build directions
 will work for any Linux-like shell environment (for example Ubuntu, macOS,
 FreeBSD etc). However, mlpack is in the repositories of many Linux distributions 
 and so it may be easier to use the package manager for your system.  For example, 
-on Ubuntu, you can install mlpack with the following command:
+on Ubuntu, you can install mlpack library and command-line executbles(eg. mlpack_pca, mlpack_kmeans etc.) with the following command:
```

Comment:
Can you wrap this line since it is longer than 80 characters please?  :)

Also it should read `the mlpack library`, `eg.` should be `e.g.`, and `executbles` should be `executables`.  (Sorry for being pedantic.)

These comments also apply to the other place you've made this change in `build.hpp`. :+1:

Reply:
No ..no its fine thanks for the change sir I come earlier from linux kernel background(i work in my company as linux developer) and I don't know if you have run into any of the maintainers of linux kernel they will easily blast you openly in front of everyone for minor details. I am suprised how I was not careful to check all this. Sorry for the  inconvenience let me correct it.